src/test/java/co/rsk/federate/FedNodeRunnerTest.java

```diff
@@ -189,6 +192,36 @@ void test_with_hsm_v1_config() throws Exception {
         assertNull(bookkeepingService);
     }
 
+    @Test
+    void test_with_hsm_v3_config_shouldFail() throws Exception {
```

Comment:
nit: 
```suggestion
    void hsm_v3_config_shouldFail() throws Exception {
```
i think no need for the `test_` prefix